.vscode/tasks.json

```diff
@@ -8,50 +24,20 @@
       "detail": "Activate conda environment",
       "group": "build"
     },
-    {
-      "type": "cmake",
-      "label": "CMake: configure",
-      "command": "configure",
-      "problemMatcher": [],
-      "detail": "CMake template configure task",
-      "group": "build"
-    },
     {
       "type": "cmake",
       "label": "CMake: build",
       "command": "build",
-      "targets": ["all"],
+      "targets": [
+        "all"
+      ],
+      "preset": "${command:cmake.activeBuildPresetName}",
       "group": {
         "kind": "build",
         "isDefault": true
       },
       "problemMatcher": [],
       "detail": "CMake template build task"
-    },
-    {
-      "type": "cmake",
-      "label": "CMake: test",
-      "command": "test",
-      "problemMatcher": [],
-      "detail": "CMake template test task",
-      "group": "build"
-    },
-    {
-      "type": "cmake",
-      "label": "CMake: clean",
-      "command": "clean",
-      "problemMatcher": [],
-      "detail": "CMake template clean task",
-      "group": "build"
-    },
-    {
-      "type": "cmake",
-      "label": "CMake: clean rebuild",
-      "command": "cleanRebuild",
-      "targets": ["all"],
-      "problemMatcher": [],
-      "detail": "CMake template clean rebuild task",
-      "group": "build"
```

Comment:
any particular reason for removing all these?

Reply:
They are available by default in the cmake extension:
press `ctrl + shift + p` to open command in vscode. type `cmake` and see these appear there

All this stuff will get documented in the handbook later